tests/cephadm/test_os_tuning_profile.py

```diff
@@ -62,16 +83,30 @@ def run(ceph_cluster, **kw):
     mount = "/tmp:/tmp"
     if config.get("command") in ("apply", "re-apply"):
         nodes = config.get("specs", {}).get("placement", {}).get("hosts")
-        hosts = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]
+        if "hostx" in nodes:
+            hosts = ["NoHost1", "NoHost2"]
+        else:
+            hosts = [ceph_cluster.get_nodes()[int(node[-1])].hostname for node in nodes]
         config["specs"]["placement"]["hosts"] = hosts
         spec_file = generate_tuned_profile_spec(installler_node, config.get("specs"))
         result = CephAdm(installler_node, mount).ceph.orch.tuned_profile.apply(
             spec_file
         )
-        if config.get("apply_result") != result:
+        if config.get("result") not in result:
```

Comment:
Are we matching substring from command result here ? Is this enough verification ?

Reply:
Hi @vamahaja 

For negative scenarios error message updating with other information as well like podman images details , keyring etc

So we are directly verifying the error message from output.
Example: https://github.com/red-hat-storage/cephci/pull/2047/files#diff-58eea54ceb47781217717b0a483e2edf5d7aee7e19a72c74d6c94c8f064226f8R357

Reply:
+1